experimental/packages/opentelemetry-sdk-metrics-base/src/aggregator/Sum.ts

```diff
@@ -37,29 +44,43 @@ export class SumAccumulation implements Accumulation {
 export class SumAggregator implements Aggregator<SumAccumulation> {
   public kind: AggregatorKind.SUM = AggregatorKind.SUM;
 
-  createAccumulation() {
-    return new SumAccumulation();
+  constructor (public monotonic: boolean) {}
```

Comment:
Maybe it's worth an options object like `{ monotonic: boolean }` in case we want to add additional configs in the future? Or do you think we're not likely to add more?

Reply:
This is an internal interface (as for now) so I'd find the flattened pattern might be more intuitive.

Reply:
ok thats fine